js/build.js

```diff
@@ -1633,7 +1635,13 @@ Fliplet.Widget.instance('chat', function(data) {
   function sendMessage($element) {
     var $parentHolder = $element.parents('.chat-input-controls');
     var $holder = $element.parents('.input-second-row');
-    var text = $messageArea.val().trim();
+    var text = DOMPurify.sanitize($messageArea.val().trim());
+
+    if (!text) {
```

Comment:
@OleksandrLavrynenko It seems that the lines underneath:

```js
      if (!$.trim(text).length && (typeof files === 'undefined' || files === '')) {
        changeUIOnMessageSent($parentHolder, $holder);
        return;
      }
```

...is already handling the scenario of what happens when `text` is empty. Can you explain how this `if (!test)` clause you added changes the functionality or UX?

Reply:
@tonytlwu Yes, you are right! Initially, when I tested, empty messages were sent to the seser several times. I checked again and it doesn't happen
I will fix this and send it for testing again.